crates/util/src/paths.rs

```diff
@@ -224,7 +300,10 @@ impl PathWithPosition {
         let maybe_file_name_with_row_col = path.file_name().unwrap_or_default().to_string_lossy();
         if maybe_file_name_with_row_col.is_empty() {
             return Self {
-                path: Path::new(s).to_path_buf(),
+                // TODO:
+                // If `s = "text_file.txt"` some relative file path, is it okay
+                // if we trimmed the path here?
```

Comment:
I do not understand this TODO: if the path is relative, it's not canonicalized and should not start with `\\?\`, so there's nothing to trim and it's ok?

Reply:
Apologies for the outdated comment—I forgot to remove it.